tests/data_context/test_get_data_context.py

```diff
@@ -272,24 +271,6 @@ def test_get_context_with_mode_equals_cloud_returns_cloud_data_context(
     assert isinstance(context, CloudDataContext)
 
 
-@pytest.mark.parametrize("ge_cloud_mode", [True, None])
-@pytest.mark.cloud
-def test_cloud_context_include_rendered_content(
```

Comment:
Do we need some cloud test to ensure we are getting rendered content?

Reply:
This is just referring to the top level configuration (which doesn't exist anymore) - rendered content should exist on each object as attributes.